lib/standard/string.nit

```diff
@@ -844,6 +844,11 @@ abstract class FlatText
 	# Real items, used as cache for to_cstring is called
 	private var real_items: nullable NativeString = null
 
+	# Returns a char* starting at position `index_from`
+	#
+	# NOTE: Not for use in extern methods, unsafe
+	fun fast_cstring: NativeString is abstract
```

Comment:
Can you tell us why not use it from C? it is awfully tempting! Is it any safer to use it from Nit?